src/metatensor/models/cli/train_model.py

```diff
@@ -92,6 +92,11 @@ def train_model(options: DictConfig) -> None:
         necessary options for dataset preparation, model hyperparameters, and training.
     """
 
+    # This gives some accuracy improvements. It is very likely that
+    # this is just due to the preliminary composition fit in the SOAP-BPNN.
+    # TODO: investigate
+    torch.set_default_dtype(torch.float64)
```

Comment:
Shouldn't we rather set this explicit for tensors where we need this?

Reply:
Personally, until we give the user control over this option, I would use `float64` because it avoids some accuracy issues. Then, when we give the user control, we can decide on a sensible default